src/legacy/server/config/schema.js

```diff
@@ -111,7 +111,8 @@ export default () => Joi.object({
       keyPassphrase: Joi.string(),
       certificateAuthorities: Joi.array().single().items(Joi.string()).default([]),
       supportedProtocols: Joi.array().items(Joi.string().valid('TLSv1', 'TLSv1.1', 'TLSv1.2')).default(['TLSv1.1', 'TLSv1.2']),
-      cipherSuites: Joi.array().items(Joi.string()).default(cryptoConstants.defaultCoreCipherList.split(':'))
+      cipherSuites: Joi.array().items(Joi.string()).default(cryptoConstants.defaultCoreCipherList.split(':')),
+      clientAuthentication: Joi.any().description('This key is handled in the new platform ONLY'),
```

Comment:
__note:__ this removes a blocker status from https://github.com/elastic/kibana/pull/42818 (at least for PKI auth provider)